foodplanner/lib/pages/settings/settings.dart

```diff
@@ -105,6 +106,12 @@ class _SettingsPage extends State<Settings> {
               SizedBox(width: 10),
             ],
           ),
+          'ctaFunction': () {
+            Navigator.push(
+              context,
+              MaterialPageRoute(builder: (context) => AdministrateChildren()),
+            );
```

Comment:
tage op til næste møde: skifte fra navigator.push til GoRouter så vi ikke laver layered widgets og vi kan få security indover. Navigator giver god mening når vi vil have et popup.

Reply:
Aight men er den her så resolved eller?

Reply:
for now yes